src/fidesops/service/connectors/query_config.py

```diff
@@ -366,6 +366,69 @@ def dry_run_query(self) -> Optional[str]:
         return None
 
 
+class MicrosoftSQLServerQueryConfig(SQLQueryConfig):
+    """
+    Generates SQL valid for SQLServer. This way of building queries should also work for every other connector,
+    but SQLServer is separated out for 2 reasons:
+
+    1. This increases code complexity for building queries
+    2. Lack of thorough testing framework to ensure all connectors will in fact work using this code
+    """
+
+    @staticmethod
+    def format_clause_for_query_in(field_name: str, query_data_names: List[str]) -> str:
+        """Returns clauses in a format they can be added into SQL queries."""
+        in_clause = ", ".join(query_data_names)
+        return f"{field_name} IN ({in_clause})"
```

Comment:
Re-implementing https://github.com/ethyca/fidesops/pull/157/files#diff-8287ffddccc208436200313a2d60459cccd653f2fe1c303754fb11db4c090785R250-R256 would still be better for consistency — did we think about flattening `in_clause` outside of the method and passing it in? That way the interface could stay the same